app/utils/isRouteAuthorized.ts

```diff
@@ -7,6 +7,7 @@ const pagesAuthorization = [
       '/analyst',
       '/applicantportal',
       '/analyst/request-access',
+      '/error-500',
```

Comment:
```suggestion
      '/500',
```

Reply:
I don't think this is needed actually, given that our 500 page doesn't use `withRelayOptions` anymore

Reply:
Nice catch, I've removed it altogether here:

https://github.com/bcgov/CONN-CCBC-portal/pull/838/commits/a00f745608bf1b6bb66c3c69540c8cfa9d4693af